www/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for www project.
+
+Generated by 'django-admin startproject' using Django 4.0.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+import os
+
+import dotenv
+
+dotenv.load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+SECRET_KEY = os.environ["DJANGO_SECRET_KEY"]
+
+DEBUG = os.environ["DJANGO_DEBUG"] == "True"
+
+ALLOWED_HOSTS = []
```

Comment:
```suggestion
ALLOWED_HOSTS = [`*`]
```
This is not "safe", but it won't run in non-debug mode otherwise, at least not until we get a domain name to host it from.